.github/workflows/check.yml

```diff
@@ -14,9 +14,6 @@ on:
         type: boolean
         required: false
         default: false
-env:
-  CARGO_TERM_COLOR: always
-  RUST_BACKTRACE: 1
```

Comment:
What is the drawback of setting `RUST_BACKTRACE`? In case we have a hard-to-reproduce unit test failure on CI, wouldn't it be good to have a stack trace?

Reply:
Ah yes, I meant to add that to the check step.